### ISSUE TITLE
Old Opencore Version (No Support for MacOS Monterey)

### ISSUE THREAD

**Issue by avneesh-go** [AniKulkarn/Hackintosh-ThinkPad-E14#30]
Hi AniKulkarn, The Opencore version of this EFI folder is 0.6.8. Apple has announced MacOS Monterey which has support starting from Opencore 0.7.0. Please Update the OC folder for better performance and Better Support for Monterey.

**Comment by AniKulkarn** [comment]
Yes. I definitely will update the OC version and also update the macOS version once I observe the stability of other Hackintoshes running Monterey.

**Comment by avneesh-go** [comment]
hello, I have created an EFI folder for E14 Gen1 with all working (Wifi,Bluetooth) and OpenCore 0.7.3 and updated all kexts and With Macbook Pro 16,3 SMBIOS as its very similar - Link for Download is https://drive.google.com/file/d/1hXNDxx2JUZxWP3nrwGtIKcV5lXkRusNu/view?usp=sharing

**Comment by avneesh-go** [comment]
Hello, I Have created an EFI folder for E14 gen 1 with all kexts updated and with opencore 0.7.3 with everything except Boot Chime working. Link is - https://1drv.ms/u/s!Aq8vmrQDnxgKeHaOHg21AklbrJw?e=GISIth

**Comment by AniKulkarn** [comment]
Thanks @avneesh-go I'll try to update this repository soon!

**Comment by AniKulkarn** [comment]
And is HDMI working too? 😃😃😃

**Comment by flying434** [comment]
> Hello, can you share your EFI, 455705397@qq.com, thank you

**Comment by flying434** [comment]
> 您好，我已经为 E14 gen 1 创建了一个 EFI 文件夹，其中所有 kexts 都已更新，并且 opencore 0.7.3 中除了 Boot Chime 之外的所有内容都可以正常工作。链接是 - https://1drv.ms/u/s!Aq8vmrQDnxgKeHaOHg21AklbrJw?e=GISIth

Hello, can you share your EFI, 455705397@qq.com, thank you
